StRoot/StFcsWaveformFitMaker/PeakAnaVirtualPainter.h

```diff
@@ -0,0 +1,70 @@
+/*
+Author: David Kapukchyan
+@[March 9, 2022]
+> Virtual interface for PeakAnaPainter. This is needed since the non-virtual class runs into "cross-dependency" issues during compilation. The idea is that a virtual interface will get rid of this problem.
+> The design is inspired/copied from ROOT's TVirtualHistPainter.
+
+PeakAnaPainter abstract base class
+*/
+
+#ifndef STROOT_STFCSWAVEFORMFITMAKER_PEAKANAVIRTUALPAINTER_H_
+#define STROOT_STFCSWAVEFORMFITMAKER_PEAKANAVIRTUALPAINTER_H_
```

Comment:
Using this example, it should just be *PEAKANAVIRTUALPAINTER_H*? I ask because I was following the include guard style in StFcsWaveformFitMaker.h. Should that be changed to just *STFCSWAVEFORMFITMAKER_H*?